src/components/Button/Button.js

```diff
@@ -18,16 +18,22 @@ const { defaultProps, propTypes } = CarbonButton;
 
 // TODO: V3 - Remove deprecated props `largeText`.
 
-const Button = ({ className, largeText, size, ...other }) => {
+const Button = ({ className, largeText, size, kind, ...other }) => {
   const isSize = value => size === value;
   const isLarge = isSize('large') || largeText || isSize('lg') || isSize('xlg');
 
+  const isKind = value => kind === value;
+  const isGhostDanger = isKind('ghost-danger');
```

Comment:
- [ ] The `isKind` method is redundant here and can be removed if it's not being used elsewhere:

```suggestion

const isGhostDanger = kind === 'ghost-danger';
```

Reply:
Thanks for this, you are absolutely right.